samples/artifacts/EventReliability/Logging.siddhi

```diff
@@ -24,27 +24,28 @@ Testing the Sample:
 Viewing the Results:
     See the input and respective output on the console similar to the following (timestamp will be different).
 
-        INFO {io.siddhi.core.query.processor.stream.LogStreamProcessor} - Logging: Sample Event :, StreamEvent{ timestamp=1568624229375, beforeWindowData=null, onAfterWindowData=null, outputData=[item1, 10.0], type=CURRENT, next=null}
-        INFO {io.siddhi.core.query.processor.stream.LogStreamProcessor} - Logging: Sample Event :, StreamEvent{ timestamp=1568624229375, beforeWindowData=null, onAfterWindowData=null, outputData=[item1, 10.0], type=CURRENT, next=null}
-        INFO {io.siddhi.core.query.processor.stream.LogStreamProcessor} - Logging: Sample Event :
-        INFO {io.siddhi.core.query.processor.stream.LogStreamProcessor} - Logging: StreamEvent{ timestamp=1568624229375, beforeWindowData=null, onAfterWindowData=null, outputData=[item1, 10.0], type=CURRENT, next=null}
-        INFO {io.siddhi.core.query.processor.stream.LogStreamProcessor} - Logging: Sample Event :, StreamEvent{ timestamp=1568624229375, beforeWindowData=null, onAfterWindowData=null, outputData=[item1, 10.0], type=CURRENT, next=null}
+        INFO {io.siddhi.core.query.processor.stream.LogStreamProcessor} - Logging: Sample Event :, StreamEvent{ timestamp=1569913910659, beforeWindowData=null, onAfterWindowData=null, outputData=[item1, 10.0], type=CURRENT, next=null}
+        WARN {io.siddhi.core.query.processor.stream.LogStreamProcessor} - Logging: Sample Event :, StreamEvent{ timestamp=1569913910659, beforeWindowData=null, onAfterWindowData=null, outputData=[item1, 10.0], type=CURRENT, next=null}
+        ERROR {io.siddhi.core.query.processor.stream.LogStreamProcessor} - Logging: Sample Event :, StreamEvent{ timestamp=1569913910659, beforeWindowData=null, onAfterWindowData=null, outputData=[item1, 10.0], type=CURRENT, next=null}
+        INFO {io.siddhi.core.query.processor.stream.LogStreamProcessor} - Logging: StreamEvent{ timestamp=1569913910659, beforeWindowData=null, onAfterWindowData=null, outputData=[item1, 10.0], type=CURRENT, next=null}
+
+
 */
 
 define stream EventStream(symbol string, price float);
 
--- This will log as INFO with the message "Sample Event :" + EventStream:events.
+-- This will log the output events with a priority of "INFO" and prefix "Sample Event :"
 from EventStream#log("INFO", "Sample Event :", true)
```

Comment:
```suggestion
from EventStream#log("INFO", "Logging at info level :", true)
```
Have debug, info and warn. Debug level will not be printed and add on how tochange editor log levels to get the log